lib/Session.h

```diff
@@ -491,7 +491,11 @@ public slots:
 private slots:
     void done(int);
 
-//  void fireZModemDetected();
```

Comment:
Although technical functionaltiy should not be affected, it's better to avoid changing slots into ordinary functions.